services/base-images/base-kernel-javascript/Dockerfile

```
@@ -27,9 +27,12 @@ COPY ./runnable-shared/runner/requirements* /orchest/services/base-images/runnab
 COPY ./lib/python /orchest/lib/python
 COPY ./orchest-sdk /orchest/orchest-sdk
 RUN chown $NB_USER -R /orchest/orchest-sdk \
-    && chown $NB_USER -R /orchest/lib
+    && chown $NB_USER -R /orchest/lib && \
+    echo -e "jovyan\njovyan" | passwd jovyan
 
+# Exclude hostname from default prompt
 USER $NB_USER
+RUN echo "export PS1='\[\033[01;32m\]\u\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '" >> ~/.bashrc
```

To reduce the number of layers in the Dockerfile (and use consistent way of breaking `RUN` commands over multiple lines):
```suggestion
RUN chown $NB_USER -R /orchest/orchest-sdk \
    && chown $NB_USER -R /orchest/lib \
    && echo -e "jovyan\njovyan" | passwd jovyan \ 
    # Exclude hostname from default prompt
    && echo "export PS1='\[\033[01;32m\]\u\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '" >> /home/$NB_USER/.bashrc

USER $NB_USER
```